### ISSUE TITLE
Accessibility: Obtain Section 508 Compliance Certification for get.gov

### ISSUE THREAD

**Issue by PaulKuykendall** [cisagov/get.gov#356]
### Issue description

As part of the quarterly compliance testing per the [.gov Accessibility Plan](https://docs.google.com/document/d/1Rt6XN7yZKvdA3A2KRkv54SzmkjQ4jOb6Xknhd0mOvKE/edit?pli=1), request certification from OAST, remediate as neccessary, to obtain an updated Compliance Determination Form (CDF) indicating successful compliance.

Additional Context (optional)
https://get.gov/
https://get.gov/domains/
https://get.gov/help/
https://get.gov/blog/
https://get.gov/about/

### Acceptance criteria

- [x] Request Submitted to OAST and ServiceNow ticket number linked here.
- [ ] CDF has been received from OAST, indicating satisfactory compliance status.
- [ ] CDF has been saved in program compliance documentation folder.
- [ ] Update Accessibility Compliance plan, as necessary and appropriate.

### Links to other issues

ServiceNow ticket: ICTR0044529
https://snow.dhs.gov/oast?id=dhs_requests

Partially Satisfies:
* cisagov/manage.get.gov#2207 

Related to:
* This ticket
* manage.get.gov ticket
* manage.get.gov/admin ticket

### Prioritization Comments
**WCAG Conformance Level:** 
**WCAG Standard that applies:** 
**Critical Failure if Not Met?:**  
**Prioritization:** 
**Reason:**

**Comment by katypies** [comment]
We likely need to re-up this work, but I'm going to call this one complete.
